src/modules/commander/HealthAndArmingChecks/checks/estimatorCheck.cpp

```diff
@@ -798,24 +799,6 @@ void EstimatorChecks::setModeRequirementFlags(const Context &context, bool pre_f
 	}
 
 	failsafe_flags.angular_velocity_invalid = angular_velocity_invalid;
-
-
-	// gps
-	if (vehicle_gps_position.timestamp != 0) {
-		bool time = (now < vehicle_gps_position.timestamp + 1_s);
-
-		bool fix = vehicle_gps_position.fix_type >= 2;
-		bool eph = vehicle_gps_position.eph < _param_com_pos_fs_eph.get();
-		bool epv = vehicle_gps_position.epv < _param_com_pos_fs_epv.get();
```

Comment:
This was the only place where `com_pos_fs_epv` was used, we can get rid of the parameter

Reply:
Good call, addressed in 3e5f23c0fddaeb1dac75aa3060cc417c06490593